src/Common/Constants.ts

```diff
@@ -131,6 +130,10 @@ export class Features {
   public static readonly enableSDKoperations = "enablesdkoperations";
 }
 
+export class Flights {

```

Comment:
>Flights [](start = 13, length = 7)

Can you please add a comment here that all flights from the framework will be lowercase?